includes/query_processor.inc

```diff
@@ -240,7 +247,7 @@ class IslandoraSolrQueryProcessor {
     $default_sort = (variable_get('islandora_solr_facet_max_limit', '20') <= 0 ? 'index' : 'count');
 
     $facet_sort_array = array();
-    foreach (array_merge($facet_array, $facet_dates) as $key => $value) {
```

Comment:
Feels like something is missing here? i see foreach happens on facet_array now but i don´t see any changed logic that supports that change further up.  I mean https://github.com/whikloj/islandora_solr_search/blob/5031f4ff93de89464cd3ec0e69831ee2a0dc9a04/includes/query_processor.inc#L201 is still there.

Reply:
See I added this back in [PR-300](https://github.com/Islandora/islandora_solr_search/pull/300) and I realized here that `$facet_dates` are already in `$facet_array` and this merge actually doesn't do anything.

Reply:
Yeah, makes sense, but looks like another ticket to me. Not saying it is not good to fix since you are here, but good to document when/why this change happened somewhere. So $facet_dates is empty? Not needed at all?

Reply:
Updated with new ticket to resolve this https://jira.duraspace.org/browse/ISLANDORA-2271